.docker/build.Dockerfile

```
@@ -0,0 +1,52 @@
+# Define ARG we use through the build
+ARG VERSION=unstable
```

Either you must pass stable as argument in the workflows or change it here to stable
```suggestion
ARG VERSION=stable
```

---

Still unstable and should be stable.